db/health/migrate/20230322172802_health_db_load.rb

```diff
@@ -0,0 +1,12 @@
+class HealthDbLoad < ActiveRecord::Migration[6.1]
```

Comment:
Should there be a note about the migration timestamp in the file name? I'm assuming that when we next roll this up, this migration needs to be replaced with a new one with the timestamp of whatever the newest is then?

Reply:
We actually shouldn't need to change the file, just update the associated structure file.  These will only ever get run if you're rebuilding in DEV or if you're launching a new installation.